lib/internal/test_runner/runner.js

```diff
@@ -426,11 +426,10 @@ function watchFiles(testFiles, opts) {
   });
   if (opts.signal) {
     kResistStopPropagation ??= require('internal/event_target').kResistStopPropagation;
-    opts.signal.addEventListener(
-      'abort',
-      () => opts.root.postRun(),
-      { __proto__: null, once: true, [kResistStopPropagation]: true },
-    );
+    opts.signal.addEventListener('abort', () => {
```

Comment:
This can use `addAbortListener` (and we could then drop the lazy require of `kResistStopPropagation`:

```suggestion
	kResistStopPropagation('abort', () => {
```